fix TestDropCurrentStateRunningTask.testDropCurrentStateRunningTask

LOG 2899
>2020-10-10T05:06:58.6016712Z [ERROR] Tests run: 1211, Failures: 1, Errors: 0, Skipped: 1, Time elapsed: 5,297.662 s <<< FAILURE! - in TestSuite
2020-10-10T05:06:58.6019017Z [ERROR] testDropCurrentStateRunningTask(org.apache.helix.integration.task.TestDropCurrentStateRunningTask)  Time elapsed: 62.391 s  <<< FAILURE!
2020-10-10T05:06:58.6056240Z org.apache.helix.HelixException: Fail to stop the workflow/queue testDropCurrentStateRunningTask with in 60000 milliseconds.
2020-10-10T05:06:58.6059726Z 	at org.apache.helix.integration.task.TestDropCurrentStateRunningTask.testDropCurrentStateRunningTask(TestDropCurrentStateRunningTask.java:152)
2020-10-10T05:06:58.6063378Z 
2020-10-10T05:06:58.9922822Z [ERROR] Failures: 
2020-10-10T05:06:58.9928845Z [ERROR]   TestDropCurrentStateRunningTask.testDropCurrentStateRunningTask:152 » Helix Fa...
2020-10-10T05:06:58.9930560Z [ERROR] Tests run: 1211, Failures: 1, Errors: 0, Skipped: 1

--- a/helix-core/src/test/java/org/apache/helix/integration/task/TestDropCurrentStateRunningTask.java
+++ b/helix-core/src/test/java/org/apache/helix/integration/task/TestDropCurrentStateRunningTask.java
@@ -149,7 +149,7 @@ public class TestDropCurrentStateRunningTask extends TaskTestBase {
                             .get(currentStatePathP1, new Stat(), AccessOption.PERSISTENT) == null),
                     TestHelper.WAIT_DURATION));
 
-    _driver.waitToStop(workflowName, TestHelper.WAIT_DURATION);
+    _driver.stop(workflowName);
   }
 
   @Test(dependsOnMethods = "testDropCurrentStateRunningTask")

